lib/dgram.js

```diff
@@ -60,14 +60,14 @@ function newHandle(type) {
 }
 
 
-exports._createSocketHandle = function(address, port, addressType, fd) {
+exports._createSocketHandle = function(address, port, addressType, fd, flags) {
   // Opening an existing fd is not supported for UDP handles.
   assert(typeof fd !== 'number' || fd < 0);
 
   var handle = newHandle(addressType);
 
   if (port || address) {
-    var err = handle.bind(address, port || 0, 0);
+    let err = handle.bind(address, port || 0, flags);
```

Comment:
It's a bit inconsistent to change `var` to `let` here, then use `var` in newly added code below.